template/osfamilymap.yaml

```diff
@@ -10,6 +10,11 @@
 # you will need to provide at least an empty dict in this file, e.g.
 # osfamilymap: {}
 ---
+{% if grains.os == 'MacOS' %}
+  {{ set macuser = salt['cmd.run']("stat -f '%Su' /dev/console") }}
+  {{ set macgroup = salt['cmd.run']("stat -f '%Sg' /dev/console") }}
+{% endif %}
+---
```

Comment:
Apologies, my examples could have been clearer:

* That `---` was to specify the exact place, not to be re-included.
* Also, we're maintaining consistency of Jinja whitespace control by using `{%- ... %}` throughout this formula.
* Finally, do we need the duplicate, unused `macuser` variable at the current time?  We can add it when it becomes time to use it.

In other words, could we have this section like this instead:

```sls
{%- if grains.os == 'MacOS' %}
{%-   set macgroup = salt['cmd.run']("stat -f '%Sg' /dev/console") %}
{%- endif %}
```

Reply:
And if `macuser` is required due to the situation mentioned below, then there's no need to duplicate the `cmd.run`.  Simply use the `macgroup` variable.  Or vice versa.

Reply:
Sure we can leave it out. 
If `macuser` (or more generically `sysuser`) is needed we can add this.  The second `cmd.run` would be needed as command to detect `user` and `group` differs.

Reply:
https://github.com/saltstack/salt/issues/53325